brave/src/test/java/brave/features/log4j2_context/Log4JThreadContextTest.java

```diff
@@ -1,76 +0,0 @@
-package brave.features.log4j2_context;
-
-import brave.Span;
-import brave.Tracer;
-import brave.Tracing;
-import brave.propagation.CurrentTraceContext;
-import brave.propagation.TraceContext;
-import org.apache.logging.log4j.ThreadContext;
-import org.junit.Test;
-
-import static org.assertj.core.api.Assertions.assertThat;
-
-public class Log4JThreadContextTest {
```

Comment:
we don't need this test as we ship log4j2 in the same repo. we would if we didn't